pipelines/skylab/multiome/Multiome.wdl

```diff
@@ -25,33 +26,56 @@ workflow Multiome {
         Boolean ignore_r1_read_length = false
         String star_strand_mode = "Forward"
         Boolean count_exons = false
-        File gex_whitelist = "gs://gcp-public-data--broad-references/RNA/resources/arc-v1/737K-arc-v1_gex.txt"
         String? soloMultiMappers
 
         # ATAC inputs
         # Array of input fastq files
         Array[File] atac_r1_fastq
         Array[File] atac_r2_fastq
         Array[File] atac_r3_fastq
-        
         # BWA tar reference
         File tar_bwa_reference
         # Chromosone sizes 
         File chrom_sizes
         # Trimadapters input
         String adapter_seq_read1 = "GTCTCGTGGGCTCGGAGATGTGTATAAGAGACAG"
         String adapter_seq_read3 = "TCGTCGGCAGCGTCAGATGTGTATAAGAGACAG"
-        # Whitelist
-        File atac_whitelist = "gs://gcp-public-data--broad-references/RNA/resources/arc-v1/737K-arc-v1_atac.txt"
 
         # CellBender
         Boolean run_cellbender = false
 
     }
 
+    # Determine docker prefix based on cloud provider
+    String gcr_docker_prefix = "us.gcr.io/broad-gotc-prod/"
+    String acr_docker_prefix = "dsppipelinedev.azurecr.io/"
+    String docker_prefix = if cloud_provider == "gcp" then gcr_docker_prefix else acr_docker_prefix
+
+    # Define docker images
+    snap_atac_docker_image = "snapatac2:1.0.4-2.3.1-1700590229"
+
```

Comment:
Will need to update this when Nareh's warp tools PR goes threw

Reply:
Just merged the warp tools PR! The new tag is `1.0.5-2.3.2-1709230223` (see [here](https://console.cloud.google.com/gcr/images/broad-gotc-prod/us/snapatac2@sha256:56172a1e5c914c0839e55404ceeecc2876321b4f806e3475b6124836e4794e75/details?authuser=0&project=broad-gotc-prod)